### PR TITLE
Tomas Javurek - typos and mentors.md

### PR DESCRIPTION
**Pull request by TomasJavurek**
Just small typos fixed and the adding myself into mentors.md, again.

### CODE REVIEW

File: _gsocproposals/2020/proposal_RucioSupportBot.md [hunk 1]
```diff
@@ -9,29 +9,29 @@ organization:
...
+Rucio is an open source software framework that provides scientific collaborations with the functionality to organize, manage, and access their volumes of data. Data in Rucio is organized using Data Identifiers (DIDs) which have three levels of granularity viz.: files, datasets and containers, respectively. Datasets are used to organise sets of files in groups and to facilitate bulk operations such as transfers or deletions. Users are permitted to perform certain actions on the DIDs such as downloads, uploads or transfers. Several escalation levels of expert support is available for users in case of problems. When satisfying answers are not found at lower support levels, a request from a user or a group of users can be escalated to the Rucio support. Due to the vast amount of support requests, we are looking into methods to assist the support team in answering these requests. Ideally, the support would be provided by an intelligent bot able to process and understand the user’s requests and finally trigger appropriate action.
```
Comment (agheata) [inline _gsocproposals/2020/proposal_RucioSupportBot.md]: "Several escalation levels of expert support is available" -> "...levels... **are** available"

File: _gsocproposals/2020/proposal_RucioSupportBot.md [hunk 2]
```diff
@@ -9,29 +9,29 @@ organization:
...
 
 ## Milestones for GSoC Student
 
  * Input Data: 
    * Creation of text corpus or use of existing one.
-   * Extraction of emails from the support mailing lists, preferably through IMAP client.
- * Extraction of the meanings of the input data and transformations needed for further analysis, preferably with the Natural Language Toolkit (NLTK).
-   * Creation of a training set and a valdiation set from the input data. 
+   * Extraction emails from the support mailing lists, preferably through IMAP client.
+ * Extraction of the meanings of the input data and and transformations needed for further analysis, preferably with the Natural Language Toolkit (NLTK).
```
Comment (agheata) [inline _gsocproposals/2020/proposal_RucioSupportBot.md]: It seems that you have put back some of the typos that I had to remove after merging. "extraction emails" and "and and" are wrong. Maybe you should run a spell/grammar checker on the full text?

Comment (TomasJavurek) [reply _gsocproposals/2020/proposal_RucioSupportBot.md]: Interesting. I did run the spell/grammar checker, but it haven't found the double and:
https://docs.google.com/document/d/1WN8okD7Dr3rExWU4MnjSXh9fpa0KxYxkw6Muscs4GWU/edit?usp=sharing

**Comment by agheata** [comment]
@TomasJavurek it seems that you have started from your rebased master, probably after you fixed some conflicts (some wrongly). Please start from a clean copy of the current hsf repo master. It is advisable to always do changes on a branch of your local fork, so that you keep your master clean. Please run also a spell/grammar checker on the text and redo this merge request so that you add a single commit (now there are 7 for a single file changed)

**Pull request closed without merging**